views/assets/js/app.js

```diff
@@ -1,17 +1,14 @@
-import $ from 'jquery';
-// import 'bootstrap';
-// import 'popper.js';
-import 'bootstrap/js/dist/collapse';
-import DateDistance from './App/DateDistance';
-import PackageFilter from './App/PackageFilter';
+import $ from 'jquery'
+import 'bootstrap/js/dist/collapse'
+import DateDistance from './App/DateDistance'
+import PackageFilter from './App/PackageFilter'
 
 function updateTimeElements () {
-    DateDistance.calculate('time')
+  DateDistance.calculate('time')
 };
 
 $(document).ready(function() {
```

Comment:
this syntax is deprecated in recent jQuery versions. Use `$(function() {})` instead.